zebrad/tests/acceptance.rs

```diff
@@ -266,7 +268,17 @@ fn version_args() -> Result<()> {
 }
 
 #[test]
-fn valid_generated_config() -> Result<()> {
+fn valid_generated_config_test() -> Result<()> {
+    // Unlike the other tests, these tests can not be run in parallel, because
+    // they use the generated config. So parallel execution can cause port and
+    // cache conflicts.
+    valid_generated_config("start", r"Starting zebrad$")?;
+    valid_generated_config("seed", r"Starting zebrad in seed mode")?;
```

Comment:
We can use the end of line character here as well.

Reply:
We use the pattern to check if zebrad is in start or seed mode.
Either of these patterns make the seed test pass:
1. `Starting zebrad in seed mode`
2. `Starting zebrad in seed mode$`

But 1. is slightly better, because it allows future updates to the end of the line -  which we don't care about.

But we need the end of line for start, because:
3. `Starting zebrad`
4. `Starting zebrad$`

The pattern 3. also matches seed mode.

Reply:
Got it, thanks.